app/assets/stylesheets/_forms.scss

```diff
@@ -1,3 +1,15 @@
+.elements-error-summary {
+  border: 5px solid #b10e1e;
```

Comment:
At the moment the use of `govuk_frontend_toolkit` is limited to previewing content – I'd like to avoid mixing toolkit and admin template variables. I'd prefer `govuk_admin_template` to pull in the colour variables from toolkit, and make them available to admin apps.

For now I think it's best to define this as a colour variable within the app.